pkg/apis/certman/v1alpha1/certificaterequest_types.go

```diff
@@ -108,6 +108,19 @@ type CertificateRequestList struct {
 	Items           []CertificateRequest `json:"items"`
 }
 
+// PlatformSecrets defines the secrets to be used by various clouds.
+type PlatformSecrets struct {
+	// +optional
+	AWS *AWSPlatformSecrets `json:"aws,omitempty"`
+}
+
+// AWSPlatformSecrets contains secrets for clusters on the AWS platform.
+type AWSPlatformSecrets struct {
+	// Credentials refers to a secret that contains the AWS account access
+	// credentials.
+	Credentials corev1.LocalObjectReference `json:"credentials"`
```

Comment:
@joelddiaz Credentials should be of type `corev1.ObjectReference` as the credentials won't be in the same namespace as the operator. I don't think `corev1.LocalObjectReference` allows you to specify `Namespace`.

Reply:
i don't believe we plan to allow a CertificateRequest in namespace A to point to a secret in namespace B, so the namespace can be inferred from the namespace on the CertificateRequest object itself. if i'm wrong in my understanding here, please let me know.